### ISSUE TITLE
In a densely mapped area, it's hard to pan the map

### ISSUE THREAD

**Issue by mvexel** [facebook/Rapid#796]
### Description

Often you would accidentally drag a node instead.
By the way, JOSM has a warning when you drag a node more than a certain distance. This has saved me from unintended bad edits multiple times.

### Version

2.00-alpha.3.3

### What browser are you seeing the problem on? What version are you running?

Chrome v109.0

### The browser URL at the time you encountered the bug

https://mapwith.ai/rapid-v2-alpha3#background=Bing&datasets=fbRoads,msBuildings&disable_features=boundaries&map=17.16/50.73303/7.09840

**Comment by bhousel** [comment]
Yeah this is something I've wanted to improve for years, and every time it makes editing harder. 😭 

I know this isn't a perfect solution but - you can do 2-finger panning on a Mac trackpad to pan the map around.  (Browsers convert this gesture to a `wheel` event which we can respond to). 

Most Mac users have been conditioned to pan this way, but it's not universal, and also people switching from JOSM probably wouldn't know to try it. 

This is frustrating too because I really thought we were training users about "2 finger panning" in the intro walkthrough, but I just checked and it is not mentioned. 😢

**Comment by mvexel** [comment]
In JOSM you pan the map by holding RMB. I find this works well for mouse users like me. 2-finger panning on trackpad works in JOSM as well (but you have to "click" as well, not sure if that is because of a system gesture setting)

**Comment by bhousel** [comment]
In 3ac0bbe14 I just made a change so that now if you drag while holding the right mouse button, it will _only_ pan the map, not drag nodes.  I think this matches how JOSM works?

I'm going to leave the issue open for now - want to try it for a while and see if this feels better.

**Comment by bhousel** [comment]
BTW there is still an issue where if the user is dragging the map with the _right_ mouse button and releases it out of the window, we never get the pointerup and the map keeps dragging.  I don't know why - performing map drags with the _left_ mouse button doesn't seem to have this issue.

**Comment by mvexel** [comment]
> In [3ac0bbe](https://github.com/facebook/RapiD/commit/3ac0bbe14f4362a1bf2914199c72eb17fe27e0b3) I just made a change so that now if you drag while holding the right mouse button, it will _only_ pan the map, not drag nodes. I think this matches how JOSM works?

That does sound like the JOSM behavior, yes.
